tools/api-client/python/lib/bmutils.py

```diff
@@ -87,6 +87,18 @@ def wrap_load_active_games(self):
       raise ValueError, "Failed to call loadActiveGames, got: " + retval
     return self._wrap_game_list_data(retval.data)
 
+  def wrap_load_new_games(self):
+    retval = self.load_new_games()
+    if not retval.status == 'ok':
+      raise ValueError, "Failed to call loadNewGames, got: " + retval
```

Comment:
This business is an old bug --- i'll try to fix it across the board, but in the meantime, fix it in new code --- you can't print retval like this because it's an object, you instead want to print `retval.message`